packages/utils/src/index.ts

```diff
@@ -197,3 +197,7 @@ export function mapObject<T extends object, U>(
 export function assertIsTruthy<T>(val: T): asserts val is NonNullable<T> {
   if (!val) throw new Error(`expected: true, actual: ${val}`);
 }
+
+export function capitalize(val: string) {
```

Comment:
Curious if we need this. Can't we just use the different variants of `text-transform`? 

```
text-transform: uppercase;
text-transform: capitalize;
```

I don't think we should do string manipulation for display purposes

Reply:
Alternatively, we could remove and always pass a `Text` `ReactNode` with the text transformed instead of passing the string?

Reply:
I think that could be good if it's always a localised string. I don't think it's a big deal though, up to you